scalikejdbc-streams/src/main/scala/scalikejdbc/streams/StreamReadySQL.scala

```diff
@@ -20,6 +23,25 @@ case class StreamReadySQL[A] private (private[streams] val underlying: SQL[A, Ha
 
   private[streams] lazy val queryTimeout: Option[Int] = underlying.queryTimeout
 
+  /**
+   * New StreamReadySQL with modification of DBSession attributes.
+   *
+   * @param makeDBSessionStreamReady The Function to modify the db session before querying.
+   */
+  final def withDBSessionAttributesModification(makeDBSessionStreamReady: DBSession => Unit): StreamReadySQL[A] = {
```

Comment:
How about `DBSessionForceAdjuster`? The function doesn't have the feature to eventually roll back its own change. In the aspect of that, that is quite different one from `DBSessionAttributesSwitcher`.


```scala
type DBSessionForceAdjuster = DBSession => Unit
def withDBSessionForceAdjuster(adjuster: DBSessionForceAdjuster): StreamReadySQL[A] = ...
```